opac/webapp/utils.py

```diff
@@ -106,3 +133,26 @@ def create_user(user_email, user_password, user_email_confirmed):
     webapp.dbsql.session.commit()
 
     return new_user
+
+
+def generate_thumbnail(input_filename, thumbnail_path, size=(100, 100)):
+    """
+    Parâmento input_filename matrix do thumbnail.
+    Parâmento thumbnail_path caminho completo para o thumbnail, o thumbnail
+    terá o mesmo nome do arquivo original concatenado com '_thumb'.
+    Parâmentro size tupla contendo o tamanho do thumbnail, padrão (100,100)
+
+    Caso a geração seja realizado com sucesso deve retorna o path do thumbnail,
+    caso contrário None.
+    """
+    name, ext = os.path.splitext(os.path.basename(input_filename))
+    image_path = os.path.join(thumbnail_path, '%s_thumb%s' % (name, ext))
```

Comment:
Para gerar o nome da imagem, a partir do filename, estamos utilizando a função: [thumbgen_filename](https://github.com/scieloorg/opac/blob/master/opac/webapp/admin/custom_fields.py#L23) 

É importante utilizar as mesmas funções para gerar os nomes, porque os "signals" vão procurando estes arquivos para apagar do filesystem quando o registro for removido. então não podemos ter nomes diferentes, tampouco é bom ter duas funções que fazem a mesma coisa.